### DISCUSSION TITLE
How do I get a new BMS into the main build?

### DISCUSSION THREAD

**Discussion by peterohman** [Louisvdw/dbus-serialbattery#504] [Q&A] [answered]
I've written a driver to support HLPdata BMS4S, a BMS developed for usage in boats. How do I get it into the main build?

**Comment by mr-manuel** [comment, marked as answer]
I saw, that you already wrote the drivers. If they are already working, than you can open a [pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request).

**Comment by peterohman** [comment]
I’ve opened a pull request. Do I need to do anything else?

/Peter

**Reply by mr-manuel** [reply]
No, only waiting
